docs/guide/agent-server-compatibility.asciidoc

```diff
@@ -0,0 +1,73 @@
+[[agent-server-compatibility]]
+== Agent/Server Compatibility
+
+Below is a chart that outlines the compatibility between different versions of the APM agents and the APM Server. 
+
+NOTE: APM Server version 6.5 introduced a new way for APM agents to communicate with the APM Server.
+To take advantage of new features,
+it is recommended that you update to the newest version of the applicable agent.
+
+[float]
+[[nodejs-compatibility]]
+==== Node.js Agent Compatibility
+
+[options="header"]
+|=======================================================================
+|Agent Version |APM Server Version
+|1.x |6.2-6.x
+|2.x |>= 6.5
+|=======================================================================
+
+[float]
+[[python-compatibility]]
+==== Python Agent Compatibility
+
+[options="header"]
+|=======================================================================
+|Agent Version |APM Server Version
+|2.x/3.x |6.2-6.x
+|4.x |>= 6.5
+|=======================================================================
+
+[float]
+[[ruby-compatibility]]
+==== Ruby Agent Compatibility
+
+[options="header"]
+|=======================================================================
+|Agent Version |APM Server Version
+|1.x |6.4-6.x
+|2.x |>= 6.5
+|=======================================================================
+
+[float]
+[[java-compatibility]]
+==== Java Agent Compatibility
+
+[options="header"]
+|=======================================================================
+|Agent Version |APM Server Version
+|1.x |>= 6.5
+|=======================================================================
+
+[float]
+[[go-compatibility]]
+==== Go Agent Compatibility
+
+[options="header"]
+|=======================================================================
+|Agent Version |APM Server Version
+|1.x |>= 6.5
+|=======================================================================
+
+[float]
+[[rum-compatibility]]
+==== RUM Agent Compatibility
+
+[options="header"]
+|=======================================================================
+|Agent Version |APM Server Version
+|0.x |6.3-6.4
```

Comment:
Can you also remove `0.x` here - afterwards feel free to merge this PR.

Reply:
Before deciding to remove see https://github.com/elastic/apm-server/pull/1530/files#r232686552